redfish-core/lib/log_services.hpp

```diff
@@ -804,7 +804,8 @@ inline void
             if (foundDumpEntry == false)
             {
                 BMCWEB_LOG_ERROR << "Can't find Dump Entry";
-                messages::internalError(asyncResp->res);
+                messages::resourceNotFound(asyncResp->res, dumpType + " dump",
+                                           entryID);
                 return;
```

Comment:
This should be dumpType_dumpId as in the incoming request

Reply:
entry id gives dump id as per incoming request.
this is how error looks
"error": {
"@Message.ExtendedInfo": [
{
"@odata.type": "#Message.v1_1_1.Message",
"Message": "The requested resource of type System dump named System_1 was not found.",
"MessageArgs": [
"System dump",
"System_1"
],